notebooks/official/pipelines/pipelines_intro_kfp.ipynb

```diff
@@ -859,7 +819,7 @@
         "\n",
         "At times you may want to use the REST API instead of the Python KFP SDK.  Below are examples of how to do that.\n",
         "\n",
-        "Where a command requires a pipeline ID, you can get that info from the \"Run\" column in the pipelines list— as shown below— as well as from the 'details' page for a given pipeline.  You can also see that info when you list pipeline job(s) via the API.\n",
+        "Where a command requires a pipeline ID, you can get that data from the \"Run\" column in the pipelines list as shown below, as well as from the \"details\" page for a given pipeline. You can see the whole pipeline details using the list method for pipeline jobs API.\n",
```

Comment:
Don't think you need "whole" here:

- You can see the pipeline details using the list method for pipeline jobs API